components/options_resolver.rst

```diff
@@ -46,27 +46,16 @@ check which options are set::
         {
             $mail = ...;
 
-            $mail->setHost(isset($this->options['host'])
-                ? $this->options['host']
-                : 'smtp.example.org');
-
-            $mail->setUsername(isset($this->options['username'])
-                ? $this->options['username']
-                : 'user');
-
-            $mail->setPassword(isset($this->options['password'])
-                ? $this->options['password']
-                : 'pa$$word');
-
-            $mail->setPort(isset($this->options['port'])
-                ? $this->options['port']
-                : 25);
+            $mail->setHost($this->options['host'] ?? 'smtp.example.org');
+            $mail->setUsername($this->options['username'] ?? 'user');
+            $mail->setPassword($this->options['password'] ?? 'pa$$word');
+            $mail->setPort($this->options['port'] ?? 25);
 
             // ...
         }
     }
 
-This boilerplate is hard to read and repetitive. Also, the default values of the
+This boilerplate code is boring and repetitive. Also, the default values of the
```

Comment:
"boring" seems like a strong word there, some people may code like that and feel you are judging them harshly c.f. https://github.com/symfony/diversity/issues/9

Reply:
I've removed the entire phrase to make it more concise. Thanks!